src/util/util.ts

```diff
@@ -205,3 +205,69 @@ export function areAllResults<T>(results: T[], result: T): boolean {
 export function capitalizeFirstLetter(text: string): string {
 	return text[0].toUpperCase() + text.slice(1);
 }
+
+/**
+ * Create a URL to an artist page on Last.fm.
+ * @param artist - Artist name
+ * @returns URL to the artist page
+ */
+export function createArtistURL(artist: string | null | undefined): string {
+	if (!artist) {
+		return '';
+	}
+	return `https://www.last.fm/music/${encodeURIComponent(artist)}`;
```

Comment:
I'm guessing we don't need to handle localisation here as lastfm should redirect?

Reply:
ah thats a good spot actually. No lastfm does not redirect.

Reply:
Looking at v2 - it seems web scrobbler never localized urls.
Since the main purpose here is to restore functionality i think localization of URLs should be a separate PR.
Did some simplification regardless, and fixed one pretty huge bug i found that i introduced in #3671 that caused applying an edit to not actually exit the edit state.